UtopiaCity/Common/Initializers/SportComplexInitializer.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq;
+using UtopiaCity.Common.Interfaces;
+using UtopiaCity.Models.Sport;
+using UtopiaCity.Models.Sport.Enums;
+
+namespace UtopiaCity.Common.Initializers
+{
+    public class SportComplexInitializer : ISubDbInitializer
+    {
+        public void InitializeSet(UtopiaCity.Data.ApplicationDbContext context)
```

Comment:
It is better to use `using UtopiaCity.Data` - this helps to keep code cleaner usually